Update custom domain section (#4325)
FusionAuth/fusionauth-site: astro/src/content/docs/get-started/run-in-the-cloud/cloud.mdx

Add a custom domain to your deployment by clicking the <InlineUIElement>Add Domain</InlineUIElement> button in the <InlineUIElement>Action</InlineUIElement> menu at the top of the listing page.

> <Aside type="caution">
>     Before adding a custom domain, a CNAME record targeting the durable URL for your deployment will need to be created for this domain, otherwise the form submission will fail. Additionally, if there is a CAA record for your domain, please ensure it authorizes the Amazon certificate authority by including any of the following domains - `amazon.com`, `amazontrust.com`, `awstrust.com`, or `amazonaws.com`.
> </Aside>

Reviewer: ```suggestion
<Aside type="caution">
    Before adding a custom domain, you must create a CNAME record targeting the durable URL for your deployment. If your domain has a CAA record, it must include at least one of the Amazon's certificate authority domains - `amazon.com`, `amazontrust.com`, `awstrust.com`, or `amazonaws.com`.
</Aside>
```
suggestion: rephrase to more prescriptive, shorter, active wording

Author: I dig this. Let me massage it a bit.
